public/app/features/plugins/angularDeprecation/AngularMigrationNotice.tsx

```diff
@@ -0,0 +1,78 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, Button, useStyles2 } from '@grafana/ui';
+import { LocalStorageValueProvider } from 'app/core/components/LocalStorageValueProvider';
+
+import { migrationFeatureFlags } from './utils';
+
+const LOCAL_STORAGE_KEY_PREFIX = 'grafana.angularDeprecation.dashboardMigrationNotice.isDismissed';
+
+function localStorageKey(dashboardUid: string): string {
+  return LOCAL_STORAGE_KEY_PREFIX + '.' + dashboardUid;
+}
```

Comment:
I don't think we have to make this alert dismissable, or if we do I don't think we should persist it in local storage.

For the angular deprecation warning (the yellow one above) I added persistance in local storage so that if the user closes it, it won't show again for that dashboard even after refreshing. This was done so users can dismiss the warning if they find it too annoying.

This other alert in particular is displayed only when setting the feature flag in the url manually, and it should be displayed if the user migrates the dashboard more than once, even if they dismissed the warning at some point.

WDYT?

Reply:
Yes, I agree, I'll remove it - this is mostly copy-paste from the other banner. 😅